docs_sources/erp_communication-source.adoc

```diff
@@ -34,6 +36,7 @@ Die Nachricht steht nun zum Abruf durch den Empfänger bereit, der seine Nachric
 |===
 ¦URI        ¦https://erp.app.ti-dienste.de/Communication
 ¦Method     ¦POST
+¦Berechtigt für ¦image:{FdV}[]
```

Comment:
Kannst du das noch mal prüfen?
Der Endpunkt POST /Communication ist ja an sich auch für das AVS berechtigt. Wir zeigen ja hier nur einen exemplarischen Verlauf wer was wann schickt.
Vielleicht würde sich da doch eher eine Matrix eigenen auf der man das aufzeigt?

Reply:
Vielleicht können wir das Inhaltsverzeichnis dafür nutzen?
Also in einer Tabelle die Anwendungsfälle auflisten und daneben die relevante Benutzergruppe hinzufügen?
Zum Beispiel: 

|Anwendungsfall|Nutzergruppe|
|---|---|
|Abrufen|AVS, KTR|
|Beliefern|AVS, KTR|
|Communication einstellen|AVS,FdV|

Reply:
Wird ein AVS eine GEM_ERP_PR_Communication_InfoReq-Nachricht schicken?
Ein FdV wird höchstwahrscheinlich auch eine GEM_ERP_PR_Communication_Reply-Nachricht schicken, oder? Also, GEM_ERP_PR_Communication_Reply ist für beides, aber ich vermute, GEM_ERP_PR_Communication_InfoReq ist nur für das FdV.

Reply:
Ja genau das ist meine Schwierigkeit das gut darzustellen. Die API sagt ja nur "POST /Communication", das können beide. Theoretisch können beide auch beide Profile benutzen.
Daher müssen wir gut überlegen wie wir das darstellen und benennen.

Reply:
Ich habe eine kleine Anpassung bei den Communication Calls vorgenommen. Das wird am einfachsten sein, aber ich bin nicht überzeugt, dass es ausreicht.